unattended_installer/wazuh_install.sh

```diff
@@ -404,14 +407,6 @@ function main() {
         checkNames
     fi
 
-# -------------- Prerequisites and Wazuh repo  ----------------------
```

Comment:
Recovered, this was a mistake

Reply:
But why is moved before the preliminary steps? 
If there is an error with the node names, etc, I prefer to know it before starting installing dependencies.

Reply:
I am sorry, I do not understand. Tell me if I understand correctly please.
Functions import. Line 332. Check the system tools.
Preliminary checks, Line 345. This step checks arguments and parameters.
Prerequisites and Wazuh repo, Line 375. This step installs dependencies
Preliminary steps, Line 383. It only validates checkPreviousCertificates.
I understand this would be correct.